gen3/bin/kube-lock.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+#
+# Script to lock namespace using labels on a lock ConfigMap (make sure 
+# KUBECTL_NAMESPACE is set so that g3kubectl works properly)
+
+help() {
+  cat - <<EOM
+  gen3 kube-lock lock-name owner:
+    Attempts to lock the lock lock-name in the namespace that KUBECTL_NAMESPACE 
```

Comment:
Can we add a timeout to a lock - so a lock auto-expires/unlocks - something like:
   `gen3 kube-lock lock-name owner max-age`